File: Python Code/Daily_Flash_Code_0/Week 1/18Jan/05-DailyFlash_Solutions/05-DailyFlash_Solutions/18_Jan_Solutions_Two/C++/prog4.cpp
#include<iostream>

int main(void) {
	std::cout << "First 50 even numbers are : ";
	for(int i = 0; i < 100; i++) {
		if((i + 1) % 2 == 0) 
			std::cout << i + 1 << " ";
	}
	return 0;
}

File: Python Code/Daily_Flash_Code_0/Week 1/18Jan/05-DailyFlash_Solutions/05-DailyFlash_Solutions/18_Jan_Solutions_Two/Java/prog4.java
class Demo {
	public static void main(String[] args) {
		System.out.println("First 50 odd numbers are : ");
		for(int i = 0; i < 100; i++) {
			System.out.print((i + 1) % 2 == 0 ? (i + 1) + " " :"");
		}
	}
}
